### ISSUE TITLE
Intermittent io.helidon.tracing.tests.it1.OpentraceableClientE2ETest.e2e

### ISSUE THREAD

**Issue by danielkec** [helidon-io/helidon#5081]
## Environment Details
* Helidon Version: main branch

----------

## Problem Description
```
[INFO] Running io.helidon.tracing.tests.it1.OpentraceableClientE2ETest
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.821 s <<< FAILURE! - in io.helidon.tracing.tests.it1.OpentraceableClientE2ETest
Error:  io.helidon.tracing.tests.it1.OpentraceableClientE2ETest.e2e  Time elapsed: 0.46 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: a collection with size <0>
     but: collection size was <1>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at io.helidon.tracing.tests.it1.OpentraceableClientE2ETest.e2e(OpentraceableClientE2ETest.java:114)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
```

This seems to be different issue than #4534

**Comment by tjquinno** [comment]
I got the same error as Joe.

**Comment by tomas-langer** [comment]
This should now be fixed by #5536
